makefile

```diff
@@ -47,6 +49,15 @@ consul:
 # build and release
 
 # build our binary in a container
+
+ifeq "$(TRAVIS)" "true"
```

Comment:
Sounds reasonable. But we'll need to put `CGO_ENABLED=0` back. It makes a difference with portability between glibc and musl libc:

without CGO_ENABLED=0:

```sh
$ ldd containerbuddy
        linux-vdso.so.1 (0x00007fff711bf000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f5f33092000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5f32ce9000)
        /lib64/ld-linux-x86-64.so.2 (0x000055f1a2073000)
```

with CGO_ENABLED=0:

```sh
$ ldd containerbuddy
        not a dynamic executable
```